<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmProgressBarColor</class>
 <widget class="QWidget" name="frmProgressBarColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ProgressBarColor" name="progressBarColor1"/>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSlider1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ProgressBarColor" name="progressBarColor2"/>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSlider2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ProgressBarColor" name="progressBarColor3"/>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSlider3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgressBarColor</class>
   <extends>QWidget</extends>
   <header>progressbarcolor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBarColor1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBarColor2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBarColor3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
